src/components/MDXRenderer/ExampleBlock/ExampleBlock.tsx

```diff
@@ -54,9 +56,24 @@ export const ExampleBlock: React.FC<ExampleBlockProps> = ({code, background, chi
             </div>
             {codePrepared ? (
                 <div className={b('code', {open: isOpen})}>
-                    <pre>
-                        <code className="language-tsx">{codePrepared}</code>
-                    </pre>
+                    <ClipboardArea
```

Comment:
Could you add `margin-right` to the code block for clipboard icon?

Now:
<img width="876" alt="image" src="https://github.com/gravity-ui/landing/assets/29424746/2173f218-a0a2-4047-99cc-f949ac586f2f">

After adding margin:
<img width="905" alt="image" src="https://github.com/gravity-ui/landing/assets/29424746/00d926cc-80ec-4dd8-8240-377c9c54fdf1">

Reply:
Yes. In this case, is it necessary to leave a background image?

Reply:
Do you mean background color? Yes, it would be better to leave it.

Reply:
done